app/controllers/list_responses_controller.rb

```diff
@@ -0,0 +1,30 @@
+class ListResponsesController < ApplicationController
+  def feedbacks
+    authorize :list_response, :feedbacks?
+    @feedbacks = Feedback.order(created_at: :asc).where(marked_done_by_id: nil).paginate(page: params[:page])
+  end
+
+  def hours
+    authorize :list_response, :hours?
+    @hours = Hour.all
+  end
+
+  def mark_feedback_done
+    feedback = Feedback.find(params[:id])
+    authorize :list_response, :mark_feedback_done?
+    feedback.update(marked_done_by: current_user)
+    redirect_to list_responses_feedbacks_path, notice: 'Erfolgreich erledigt markiert'
```

Comment:
"Erfolgreich *als* erledigt markiert"

Also, do we need a redirect and/or flash notice in the other `mark_*` actions?